### ISSUE TITLE
Feature Request: Custom tick box

### ISSUE THREAD

**Issue by SteveWNZ** [bmlt-enabled/bmlt-workflow#177]
Hi Nigel

Would it be possible to add a tick box on user form to display a line of text in the submissions?
With the following on the configuration page...
Tick box for if its displayed on the form, Editable text label for tick box the text is displayed on the submission?
Just needs to be visible on submissions.

The reason for this is our region provides zoom bookings for meetings maybe other regions may have uses for this too :)
Thanks
Steve

**Comment by nigel-bmlt** [comment]
Hey Steve,
I'm not quite sure I'm clear on the request. You're looking for a custom text field and a tick box that goes through with the submission?
Can you explain a little more about how the scenario works for the submitter and then the approver?

**Comment by SteveWNZ** [comment]
Hi Nigel
We have a few zoom hosing accounts to provide bookings to members for VM/HY meetings and service meetings
This would be a tick box for the submitter to inform us that they require a booking for their new VM/HY meeting. It only needs to show up a message that they want this in the workflow submissions. With the approval it does not need to be stored or cause any kind of action. Hope this makes sense feel free to contact me and thanks for all your awesome service

**Comment by SteveWNZ** [comment]
we would then make their booking, add the zoom details to the submission in workflow and then approve it

**Comment by nigel-bmlt** [comment]
hey steve - how would this differ from using the virtual meeting format and filling in the zoom meeting that way? 

<img width="814" alt="image" src="https://github.com/bmlt-enabled/bmlt-workflow/assets/99465040/c79f7dc1-8f52-49f9-9e8f-48139b98a8ec">

**Comment by nigel-bmlt** [comment]
ah ok i think i get your ask - this isnt for bmlt related changes at all, just for general purpose meeting bookings on your zoom accounts?
let me have a think about how we might be able to implement this. maybe something like a custom field with tickbox that makes it all the way through to the approval page?

**Comment by SteveWNZ** [comment]
Yes you have it, The members will not know the zoom details when they submit the form. This is just to let us know that we need to assign them a booking. Thanks Nigel Appropriate all your awesome work :)
